site/packages/aa-core/smart-account-client/middleware/index.md

```diff
@@ -0,0 +1,79 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: SmartContractAccount
+  - - meta
+    - name: description
+      content: Overview of ClientMiddleware exported by aa-core
+  - - meta
+    - property: og:description
+      content: Overview of ClientMiddleware exported by aa-core
+---
+
+# ClientMiddleware
```

Comment:
we should also add the type used in `SmartAccountClientParams` for setting these since the paymaster and data and dummy paymaster and data are merged there

Reply:
updated @moldy530